ipinfo/cmd_config.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
As a side note, for the future you can push your PRs to this main `ipinfo/cli` repository instead of your fork.